docs/design-ux/editor.md

```diff
@@ -0,0 +1,37 @@
+# Editor
+When a user has a document selected, that file is open for editing in an editor. The name of the document is displayed above the document and is editable (editing renames the document).
+
+An icon near the document's name indicates one of the following statuses:
+* the document is presented exactly as it appears on disk
+* all changes to the document have been merged into the version on disk
+* there are unsaved changes to the document
+
+At most 5s after the user completes an edit on this document, the document is saved. At most 60s after the user completes an edit on any document, a sync occurs and any open documents are reloaded from disk.
+
+On platforms that present an editor side-by-side with navigation, the navigation is collapsible.
+
+## Markdown
+At most 1s after the user completes an edit, the document is syntax-highlighted and the active style is set in accordance with the syntax highlighting at the cursor's position. The syntax highlighting makes links clickable to open the link with the user's default browser.
```

Comment:
If you're bringing up links, maybe we want to also think through embedded drawings like we saw from the other app during our call.

Could also be worth exploring the links story:
+ relative links
+ permanent links (ids)
+ be able to answer which documents link to this one